py-polars/src/polars/schema.py

```diff
@@ -199,6 +204,39 @@ def dtypes(self) -> list[DataType]:
         """
         return list(self.values())
 
+    @unstable()
+    def to_arrow(self, *, compat_level: CompatLevel | None = None) -> pa.Schema:
+        """
+        Convert the schema to a pyarrow schema.
+
+        Parameters
+        ----------
+        compat_level
+            Use a specific compatibility level
+            when exporting Polars' internal data types.
+
+        Examples
+        --------
+        >>> pl.Schema({"x": pl.String}).to_arrow()
+        x: string_view
+        """
+
+        class SchemaCapsuleProvider:
+            def __init__(self, schema: Schema, compat_level: CompatLevel) -> None:
+                self.schema = schema
+                self.compat_level = compat_level
+
+            def __arrow_c_schema__(self) -> object:
+                return polars_schema_to_pycapsule(
+                    self.schema, self.compat_level._version
+                )
+
+        return pa.schema(
+            SchemaCapsuleProvider(
+                self, CompatLevel.newest() if compat_level is None else compat_level
+            )
+        )
```

Comment:
With the existing implementation of `__arrow_c_schema` https://github.com/pola-rs/polars/blob/eba9bebf9cbf8711c4bd4f2485c9d8326aee771c/py-polars/src/polars/schema.py#L169-L176

can't you just do
```py
return pa.schema(self)
```
?